extensions/notebook/src/test/book/book.test.ts

```diff
@@ -194,6 +196,32 @@ describe('BookTreeViewProviderTests', function () {
 				equalBookItems(notebook3, expectedNotebook3);
 			});
 
+			it('sould set notebooks trusted to true on trustBook', async () => {
+				// await bookTreeViewProvider.openBook(rootFolderPath);
+				let notebook1Path = path.join(rootFolderPath, 'Book', 'content', 'notebook1.ipynb');
+				let bookTrustManager: BookTrustManager = new BookTrustManager(bookTreeViewProvider.books, appContext.apiWrapper);
+				let isTrusted = bookTrustManager.isNotebookTrustedByDefault(notebook1Path);
+				should(isTrusted).equal(false, 'Failed to set trust on trustBook');
```

Comment:
Message doesn't seem to match what's being checked here - false is just the default value